samples/ApptentiveSample/MyApplication.cs

```diff
@@ -17,10 +17,12 @@ public override void OnCreate()
         {
             base.OnCreate();
 
-            var configuration = new ApptentiveConfiguration("Your Apptentive Key", "Your Apptentive Signature");
-            configuration.SetLogLevel(ApptentiveLog.Level.Verbose);
-            configuration.SetShouldSanitizeLogMessages(false);
-            Apptentive.Register(this, configuration);
+            var configuration = new ApptentiveKit.Android.ApptentiveConfiguration("ANDROID-XAMARIN-ANDROID", "22527498ae05c9605dd746cd7c5cb33c");
+            configuration.LogLevel = Apptentive.Com.Android.Util.LogLevel.Verbose;
+            configuration.CustomAppStoreURL = "https://www.alchemer.com";
```

Comment:
The distributionName & distributionVersion are meant to be set in the plugin. The values would be "Xamarin" and the "<plugin-version>"